### PR TITLE
Mask utils

### PR DESCRIPTION
**Pull request by scaelles**

### CODE REVIEW

File: davisinteractive/utils/mask.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+def combine_masks(masks, th=0.5, method='max_per_pixel'):
+    """ Combine mask for different objects
+
+    # Arguments
+        masks: List containing a list of masks for every object. Therefore, len(masks) == number_objects
+               and len(masks[0]) == number_frames. The masks should be numpy array.
+        th: Threshold to binarize the masks. Default: 0.5
+        method: Method that specifies how the masks are fuse, the following are available:
+                - max_per_pixel(default): Computes
```
Comment (albertomontesg) [inline davisinteractive/utils/mask.py]: Missing sentence.

File: davisinteractive/utils/mask_test.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import os
+
+from PIL import Image
+import numpy as np
+
+from .mask import combine_masks
+
+
```
Comment (albertomontesg) [inline davisinteractive/utils/mask_test.py]: Good tests using some raw data to test it!

My only suggestion is to define here:
```python
TEST_DIR = os.path.join(os.path.dirname(__file__), 'masks_test')
```

And the resulting lines on the tests won't be so nested.

**Comment by albertomontesg** [comment]
Rebase the branch with the latest commits in master and then the pull request will be good to merge.

File: davisinteractive/utils/mask.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+
+def combine_masks(masks, th=0.5, method='max_per_pixel'):
+    """ Combine mask for different objects
+
+    Different methods are the following:
+
+    * `max_per_pixel`: Computes
```
Comment (albertomontesg) [inline davisinteractive/utils/mask.py]: The explanation is missing

File: davisinteractive/utils/mask_test.py (hunk 2 above)
Comment (scaelles) [reply davisinteractive/utils/mask_test.py]: Done!

File: davisinteractive/utils/mask.py (hunk 1 above)
Comment (scaelles) [reply davisinteractive/utils/mask.py]: Done!

**Comment by scaelles** [comment]
The rebase and the suggested improvements are done. If there is nothing else missing, feel free to merge the branch.

**Pull request merged**